dispatcher/backend/src/common/schemas/offliners/devdocs.py

```diff
@@ -0,0 +1,162 @@
+            "label": "ZIM name",
+            "description": "ZIM name. You can use placeholders, see "
+            "https://github.com/openzim/devdocs/blob/main/README.md. Defaults "
+            "to devdocs.io_en_{clean_slug}",
+        },
+        data_key="name-format",
+    )
+
+    title_format = String(
+        metadata={
+            "label": "ZIM title",
+            "description": "ZIM title. You can use placeholders, see "
+            "https://github.com/openzim/devdocs/blob/main/README.md. Defaults "
+            "to `{full_name} Docs`",
+        },
+        data_key="title-format",
+    )
+
+    description_format = String(
+        metadata={
+            "label": "ZIM description",
+            "description": "ZIM description. You can use placeholders, see "
+            "https://github.com/openzim/devdocs/blob/main/README.md. Defaults "
+            "to `{full_name} docs by DevDocs`",
+        },
+        data_key="description-format",
+        validate=validate_zim_description,
+    )
+
+    long_description_format = String(
+        metadata={
+            "label": "ZIM long description",
+            "description": "ZIM long description. You can use placeholders, see "
+            "https://github.com/openzim/devdocs/blob/main/README.md. Defaults "
+            "to `{full_name} docs by DevDocs`",
+        },
+        data_key="long-description-format",
+        validate=validate_zim_longdescription,
+    )
+
+    tags = String(
+        metadata={
+            "label": "ZIM Tags",
+            "description": "List of semi-colon-separated Tags for the ZIM file. "
+            " You can use placeholders, see "
+            "https://github.com/openzim/devdocs/blob/main/README.md. Defaults to"
+            "`devdocs;{slug_without_version}`",
+        }
+    )
+
+    creator = String(
+        metadata={
+            "label": "Creator",
+            "description": "Name of content creator. “DevDocs” otherwise",
+        },
+    )
+
+    publisher = String(
+        metadata={
+            "label": "Publisher",
+            "description": "Custom publisher name (ZIM metadata). “openZIM” otherwise",
+        },
+    )
+
+    output = String(
+        metadata={
+            "label": "Output folder",
+            "placeholder": "/output",
+            "description": "Output folder for ZIM file(s). Leave it as `/output`",
+        },
+        load_default="/output",
+        dump_default="/output",
+        validate=validate_output,
+    )
+
+    debug = fields.Boolean(
+        truthy=[True],
+        falsy=[False],
+        metadata={"label": "Debug", "description": "Enable verbose output"},
+    )
+
+    devdocs_frontend_url = String(
+        metadata={
+            "label": "DevDocs frontend URL",
+            "description": "Scheme and hostname for the devdocs frontend."
+            "Defaults to https://devdocs.io",
+        },
+        data_key="devdocs-frontend-url",
+    )
+
+    devdocs_documents_url = String(
```

Comment:
Should we expose this dev-only option?

Reply:
I think it might avoid to have to release the scraper should we have to update this. But it is not like this is going to change, I agree.

Reply:
Removed